src/core/pages/PatientsList/PatientCreate.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { TextInput, DateInput, RadioButtonGroupInput } from "react-admin";
+
+import { withStyles } from '@material-ui/core/styles';
+
+import CreateTemplate from "../../common/ResourseTemplates/CreateTemplate";
+
+const styles = {
+    halfWidthBlock: {
+        width: "auto !important",
+    },
+    halfWidth: {
+        display: "inline-block",
+        width: "50% !important",
+    },
+    labelBlock: {
+        '& > div': {
+            marginBottom: "0px !important",
+        },
+    },
+};
+
+/**
+ * This component returns Patient creation form
+ *
+ * @author Bogdan Shcherban <bsc@piogroup.net>
+ * @param {shape} classes
+ * @param {shape} rest
+ */
+const PatientCreate = ({ classes, ...rest }) => (
+    <CreateTemplate blockTitle="Patient" {...rest}>
+        <TextInput source="name" label="Name" fullWidth />
+        <DateInput source="dateOfBirth" label="Born" fullWidth />
+        <RadioButtonGroupInput source="gender" label="Gender" choices={[
```

Comment:
extract the choices to a constant to avoid forcing a rerender of RadioButtonGroupInput each time PatientCreate rerenders

Reply:
Done: I created separate component for form inputs and extract choices there. Styles are located in this component.